kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -329,6 +329,29 @@ public class BigQuerySinkConfig extends AbstractConfig {
       + "tables, and periodic merge flushes. Row-matching will be performed based on the contents " 
       + "of record keys.";
 
+  public static final String USE_STORAGE_WRITE_API_CONFIG = "useStorageWriteApi";
+
+  private static final ConfigDef.Type USE_STORAGE_WRITE_API_TYPE = ConfigDef.Type.BOOLEAN;
+  public static final boolean USE_STORAGE_WRITE_API_DEFAULT = true;
```

Comment:
the default behaviour changes with this one

Reply:
Thought of keeping this true till we are release ready so all tests run with this config true.